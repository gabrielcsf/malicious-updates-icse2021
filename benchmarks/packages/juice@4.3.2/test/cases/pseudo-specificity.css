span.a {
  color: red;
  background-color: red;
}
.a:first-child {
  color: blue;
  background-color: blue;
}
.a:not(div) {
  background-color: green;
  padding: 0px;
}
.a:first-child {
  padding: 10px;
}
span.b {
  padding: 20px;
}
