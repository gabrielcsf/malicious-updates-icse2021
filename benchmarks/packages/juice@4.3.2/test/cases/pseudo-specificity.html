<div>
  <span class="a b"></span>
  <span class="c"></span>
</div>