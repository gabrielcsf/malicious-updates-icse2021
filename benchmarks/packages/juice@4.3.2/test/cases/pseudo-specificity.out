<div>
  <span class="a b" style="color: blue; background-color: green; padding: 20px;"></span>
  <span class="c"></span>
</div>